src/skin/popup.html

```diff
@@ -80,5 +80,6 @@ <h1 id="report_title" class="i18n_report_title"></h1>
   <center><button id="donate" class="pbButton" style="display:block" target="blank"><span class="i18n_donate_to_eff"></span></button></center>
 </div>
 <div class='clear'></div>
+<div class='footer'><span id="version" class="i18n_version"></span></div>
```

Comment:
How does this look:
![pb_moved_to_right](https://cloud.githubusercontent.com/assets/4513790/25639456/d6e5ebf4-2f40-11e7-9575-0c4bdace8e83.png)

There were moved 5px to the right.